src/products/src/products-service/data/products.yaml

```diff
@@ -6,6 +6,7 @@
   price: 109.99
   image: "1.jpg"
   featured: true
+  currentstock: 1
```

Comment:
See my comment for product.go below about naming `current_stock` instead.

Reply:
Should the stock levels be higher than 1, at least for some products?

Reply:
I've put random numbers between 1-15